.github/workflows/ci.yml

```diff
@@ -36,6 +37,8 @@ jobs:
           - os: macOS-latest
             ghc: 8.2.2
 
+          - os: windows-latest
+            ghc: 9.2.1
```

Comment:
Let's also disable `ghc 9.2.1` for `macOS-latest`. Usually, it's enough to build only with one compiler on different operating systems just to make sure that your project compiles and tests pass. Ideally, with the latest compiler, but the latest GHC doesn't always work. I believe, GHC 8.10.7 is still the recommended version that should work almost everywhere 🤔 

So, to speed up CI and don't waste extra resources, we build `N + M` versions instead of `N * M`

Reply:
Disabled as requested. Personally I would go for `N * M` - you never can be sure 😉 and some some issues (e.g. https://github.com/Deewiant/glob/issues/40 - that was the reason why I had to disable windows 9.2.1) only appear on one platform. And I agree GHC 8.10 is at the moment still the predominant version; but I guess this will rapidly change with stackage support for 9.x.